### PR TITLE
Add argument to create_synthetic_data on whether adults are married.

### PR DESCRIPTION
**Pull request by ChristianZimpelmann**
### What problem do you want to solve?

Add argument to create_synthetic_data on whether adults are married.

### CODE REVIEW

File: src/_gettsim/synthetic.py [hunk 1]
```diff
@@ -287,8 +306,6 @@ def create_constant_across_households_variables(df, n_adults, n_children, policy
             bg_daten["bruttokaltmiete"][hh_typ_string_lookup]
         ),
         "heizkosten_m_hh": float(bg_daten["heizkosten"][hh_typ_string_lookup]),
-        "kind_unterh_anspr_m": 0.0,
```
Comment (MImmesberger) [inline src/_gettsim/synthetic.py:290]: Why did you remove this and the next line?

**Review by MImmesberger — COMMENTED** [review]
Thanks! Just one question.

File: src/_gettsim/synthetic.py (hunk 1 above)
Comment (ChristianZimpelmann) [reply src/_gettsim/synthetic.py:290]: all float input variables which are not defined in this dictionnary are set to 0.0 anyway below. So no need to include those two here.

Comment (MImmesberger) [reply src/_gettsim/synthetic.py:290]: Ah thanks, didn't know that!

**Review by MImmesberger — APPROVED** [review]
